Manual.md

```diff
@@ -1302,6 +1326,360 @@ cache during the install/removal of the package
 - `font_dirs`: which should be set to the directory where the package
 installs its fonts
 
+<a id="xbps_triggers"></a>
+### XBPS Triggers
+
+XBPS triggers is a collection of snippets of code, provided by the `xbps-triggers`
+package, that are added to the INSTALL/REMOVE scripts of packages either manually
+by setting the `triggers` variable in the template, or automatically, when specific
+conditions are met.
+
+The following is a list for each trigger available, its current status, what it does
```

Comment:
`...list of all available triggers, their status and what each of them does...` maybe?